jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfigCredentialRetriever.java

```diff
@@ -122,6 +124,8 @@ public DockerConfigCredentialRetriever(String registry, Path dockerConfigFile) {
             new String(Base64.decodeBase64(auth), StandardCharsets.UTF_8);
         String username = usernameColonPassword.substring(0, usernameColonPassword.indexOf(":"));
         String password = usernameColonPassword.substring(usernameColonPassword.indexOf(":") + 1);
+        logger.accept(
+            LogEvent.debug("Docker config auths section defines credentials for " + registryAlias));
```

Comment:
I think we need to add a test for these two debug statements?

Reply:
Added tests. I increased the level to INFO from DEBUG, as I think you usually pass `--info` for Gradle.

Reply:
Oh does that make our tests too noisy?

Reply:
I don't think it will. Maybe just a little bit more lines.